### PR TITLE
updates links and descriptions to Protect Your Data activities

### LINKED ISSUE #2392: Update links and index copy for Privacy Basics
See https://docs.google.com/document/d/1BSHbGmK8DW2f6-svDmBmmBdL5eCm87elUUNFMN9uyyg/edit#heading=h.oj12bsbwexeb

### PR DESCRIPTION
**Pull request by hannahkane**
- [X] Localization addressed - NA

Fixes #2392

**Changes proposed in this pull request:**
- updates links and index copy for Privacy Basics module
- updates two images

### CODE REVIEW

**Comment by hannahkane** [comment]
anyone have a moment to review this? @alanmoo? @gideonthomas?

**Comment by alanmoo** [comment]
Looking now

File: pages/protect-your-data.jsx [hunk 1]
```diff
@@ -24,73 +23,61 @@ var LearningObjectives = (
...
-        description: "Learners generate pass-phrases, pronounceable and random passwords based on best practices. Then they create a survey to pass along their knowledge to friends and family.",
-        link: "http://mozilla.github.io/webmaker-curriculum/ProtectingYourData/session02-secure-passwords.html"
-      }
-    ]
-  },
-  {
-    title: "Participating on the Web",
-    activities: [
+        title: "Create Secure Passwords",
+        image1x: "/img/pages/protect-your-data/pass-phrase.jpg",
```
Comment (alanmoo) [inline pages/protect-your-data.jsx]: I don't know where this image came from, but

File: pages/protect-your-data.jsx [hunk 2]
```diff
@@ -24,73 +23,61 @@ var LearningObjectives = (
...
       },
       {
         title: "Privacy Coach",
         image1x: "/img/pages/protect-your-data/img-privacycoach.png",
         image2x: "/img/pages/protect-your-data/img-privacycoach@2x.png",
-        subtitle: "Understanding privacy and open practices",
-        description: "Learners become privacy mentors to their peers by sharing their expertise through discussion or creating their own &ldquo;recipe&rdquo; documentation on privacy.",
-        link: "http://mozilla.github.io/webmaker-curriculum/ProtectingYourData/session03-privacycoach.html"
+        subtitle: "Open Practice, Protect",
+        description: "rYou will make a teaching and learning resource you can use to mentor your peers about online privacy and protecting their data while learning web literacy skills like <strong>open practice<strong> and <strong>protect</strong>.",
```
Comment (alanmoo) [inline pages/protect-your-data.jsx]: Extra r

File: pages/protect-your-data.jsx [hunk 3]
```diff
@@ -6,14 +6,13 @@ var ActivitySection = require('../components/activity-section.jsx');
...
-    <h1>Protect Your Data</h1>
+    <h1>Privacy Basics: Protect Your Data</h1>
     <section className="intro">
       <Illustration
       height={204} width={204}
-      src1x="/img/pages/protect-your-data/photo-protect-your-data.png"
-      src2x="/img/pages/protect-your-data/photo-protect-your-data@2x.png"
-      alt="">
-        <h2>Engage learners around the topic of privacy in this six-part module, developed in conjunction with Hive Toronto and the Mozilla Privacy Team.</h2>
+      src1x="/img/pages/protect-your-data/ip-tracer-badge.jpg"
```
Comment (alanmoo) [inline pages/protect-your-data.jsx]: This image gets squished

**Comment by hannahkane** [comment]
Thanks, @alanmoo. I fixed the typo and added the style. I think it looks okay. Most obvious repercussions are on /activities, but I think it's...fine. What do you think?

**Comment by alanmoo** [comment]
Good point. I think removing [this line](https://github.com/mozilla/learning.mozilla.org/blob/master/less/components/illustration.less#L55) should reduce that weirdness, though I didn't check everywhere the `<Illustration>` component might be used.

**Comment by alanmoo** [comment]
Ok, if you merge in https://github.com/hannahkane/learning.mozilla.org/pull/1, this PR should then be ready to go.

**Comment by hannahkane** [comment]
Thanks @alanmoo!

**Pull request merged**